examples/models/object_detection/faster_rcnn/train.py

```diff
@@ -18,16 +18,38 @@
 Last modified: 2022/09/27
 Description: Use KerasCV to train a RetinaNet on Pascal VOC 2007.
```

Comment:
File name wise, since this is train wih pascal voc, how about change it to train_pascal_voc.py?